### ISSUE TITLE
`python setup.py install` not working

### ISSUE THREAD

**Issue by romainvo** [epic-kitchens/epic-kitchens-100-object-masks#1]
I first installed the hand-objects-bboxes api from this repo on my environment https://github.com/epic-kitchens/epic-kitchens-100-hand-object-bboxes/ and then try to install the library for the objects masks but when running the command in my conda env, when printing `conda list` or simply running the 2nd cell of the demo notebook, it says : 
```
ModuleNotFoundError                       Traceback (most recent call last)
 in 
      5 from ipywidgets import interact, IntSlider, Layout
      6 
----> 7 from epic_kitchens.masks.io import load_detections
      8 from epic_kitchens.masks.visualisation import DetectionRenderer

ModuleNotFoundError: No module named 'epic_kitchens.masks'
```

I tried to install the objects-masks api first without installing the hand-objects-bboxes but it's not working either.

**Comment by romainvo** [comment]
I found that there was no `__init__.py` file in the epic_kitchens/masks/ directory, I tried to add one but it didn't solve the problem, should I write particular import in it ?

**Comment by willprice** [comment]
Could you try now to see if your problem is resolved?

**Comment by willprice** [comment]
Closing due to inactivity, I assume I resolved this in 146859a655cc5f410ef43fbe312467d110bd53a3 and 1c20ac253421e9ac2c3ea8dab0590effeefe59a7. If not please reopen this issue.
